elytron/WFLY-13762_ssl_context_based_on_peer_info.adoc

```diff
@@ -0,0 +1,96 @@
+= WFLY-13762 SSLContext to support delegation to alternate instances based on peer information
+:author:            Diana Vilkolakova
+:email:             dvilkola@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+As an SSLEngine is created the host and port information of the peer being accessed is available at the time the engine is requested:
+
+https://download.java.net/java/early_access/jdk11/docs/api/java.base/javax/net/ssl/SSLContext.html#createSSLEngine(java.lang.String,int)
```

Comment:
Just a minor comment but looks like this link should be updated.

Reply:
Fixed. Thank you